frontend/public/locales/en/status.json

```diff
@@ -32,10 +33,10 @@
   },
   "privacy-notice-statement": {
     "summary": "Privacy Notice Statement",
-    "collection-of-use": "The collection and use of your personal information is authorized under the Department of Health Act, the Dental Care Measures Act, the Department of Employment and Social Development Act, and other applicable laws.",
+    "collection-of-use": "The collection and use of your personal information is authorized under the <i>Department of Health Act</i>, the <i>Dental Care Measures Act</i>, the <i>Department of Employment and Social Development Act</i>, and other applicable laws.",
```

Comment:
See https://github.com/DTS-STN/canadian-dental-care-plan/pull/1736#discussion_r1640111206, which if agreed on, can be applied to other i18n values in this PR.

Reply:
FYI adding `<cite>` directly to these i18n files will render the HTML tag:
![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/55502055/2e15a8f5-f26b-4ede-8067-fe52fd1f5295)

We have to pass a `cite` component to a `Trans` to fix this. See https://github.com/DTS-STN/canadian-dental-care-plan/blob/3a7e8d67090849dc55e47e2b047ac35f1442e1dd/frontend/app/routes/%24lang/_public/apply/%24id/terms-and-conditions.tsx#L115

(credit to @AdamP-HRSDC-RHDCC)

Reply:
(FYI @nibivi77 I see a commit from today but `frontend/app/routes/$lang/_public/status/index.tsx` hasn't been changed to address the above)

Reply:
OOh ! Sorry I just saw this comment